### ISSUE TITLE
Onesix launcher fails to launch instance

### ISSUE THREAD

**Issue by ryanseipp** [MultiMC/MultiMC5#3190]
System Information
-----------------------------
MultiMC version: 0.6.11 - Also tested on 0.6.12-custom (Git 03dc8dd)
Operating System: Linux 5.6.15 (Arch Linux distro)
Java version: java-8-openjdk (1.8.0_252)

Summary of the issue or suggestion:
----------------------------------------------
Onesix launcher fails to start any minecraft instance, modded or vanilla, on minecraft version 1.12.2.

What should happen:
------------------------------
MultiMC should start the instance, modded or otherwise, and launch into the game window.

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1. git clone https://aur.archlinux.org/multimc5.git
2. cd multimc5 && makepkg -sri
3. multimc
4. Add vanilla Minecraft instance version 1.12.2
5. Launch

Suspected cause:
---------------------------
The method in which Minecraft or the launcher attempts to initialize the game window is failing to communicate with Xorg. Within the stack trace, we can see that the onesix launcher properly calls Minecraft's `Main` method but fails on a call to `org.lwjgl.opengl.XRandR.findPrimary(XRandR.java:326)`. This leads me to believe that lwjgl (2.9.4-nightly020150209-natives-linux) is flawed or there is some further java version mismatch or something that is outside my realm of understanding.

Logs/Screenshots:
----------------------------
https://pastebin.com/4mtNEWH2 - Vanilla Minecraft version 1.12.2
https://pastebin.com/Z1FGGcak - MC Eternal 1.3.5.3

Additional Info:
---------------------------
This is all being done on Arch Linux with i3 as the window manager. A primary display has been set in the Xorg configs, and validated using the below to no avail.
```sh
xrandr --output DisplayPort-0 --primary
```

**Comment by ryanseipp** [comment]
After some further testing I'm realizing this has more to do with the resolutions of my displays. I recently upgraded to 1440p monitors from 1080p. Before upgrading, MultiMC worked perfectly and game instances would launch. I verified this by plugging in my old monitor and the game instance launched like normal.

I attempted changing the default window size to 1920x1080 on my 1440p monitor to see if this would help, but had no success. I'm now wondering if there is some workaround for 1440p displays and lwjgl 2.9.4 or some arguments the jvm will accept that will help here.
